.github/workflows/main.yml

```diff
@@ -20,17 +20,17 @@ jobs:
     # The type of runner that the job will run on
     runs-on: ubuntu-latest
     env:
-      NODE_VERSION: 14
+      NODE_VERSION: 18
```

Comment:
not sure if Cloudflare Pages (where docs deploy to) support node 18?

Reply:
> not sure if Cloudflare Pages (where docs deploy to) support node 18?

Yeah, you’re right. It only goes up to 17. Il update that right now!